pkg/ha/stage.go

```diff
@@ -16,7 +16,7 @@ package ha
 import (
 	"context"
 	"encoding/json"
-	"fmt"
+	"github.com/pingcap/dm/pkg/terror"
```

Comment:
re-order it.

Reply:
https://github.com/pingcap/dm/pull/848/commits/c6e4bf9a5c25b7f9cdb9cd4c73ddea768e72f6f2

do you have any tools recommended to handle grouping-import 😂

Reply:
not found yet, 😢